opus/application/static_media/js/hash.js

```diff
@@ -45,6 +46,67 @@ var o_hash = {
         return hash.join("&");
     },
 
+    encodeSlugValue: function(slugValueArray) {
```

Comment:
This probably should be called encodeSlugValues since it can encode more than one value